qa/rpc-tests/wallet_persistence.py

```diff
@@ -45,6 +60,20 @@ def run_test(self):
         addresses = self.nodes[0].z_listaddresses()
         assert_true(sapling_addr in addresses, "Should contain address before restart")
 
+        def check_chain_value(pool, expected_value):
```

Comment:
```suggestion
        def check_chain_value(pool, expected_id, expected_value):
            assert_equal(pool['id'], expected_id)
            assert_equal(pool['monitored'], True)
```

Reply:
Huh, I thought I had already reviewed a version that had made these changes?

Reply:
Hmmm, you're correct. Let me see if I can figure out what happened here.

Reply:
I've restored these changes.